.github/workflows/typescript-eventsource-publish-action.yml

```diff
@@ -0,0 +1,21 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish TypeScript-EventSource SDK package to NPM registry
+on:
+  workflow_dispatch:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: cd ../../sdks/typescript/client-typescript-eventsource
```

Comment:
it won't remember where you are from step to step, each step should be self contained as it is executed in a shell.  And you want them to succeed, so cd X && npm install && npm run release 

then you will need to commit the change back, so ideally this workflow would run only when a specific situation occurs, we need to talk about what that action would be.

Reply:
i see, ok let's discuss this